ibazel/main.go

```diff
@@ -27,6 +27,7 @@ var overrideableBazelFlags []string = []string{
 
 var logToFile = flag.String("log_to_file", "-", "Log iBazel stderr to a file instead of os.Stderr")
 var noLiveReload = flag.Bool("nolive_reload", false, "Disable JavaScript live reload support")
+var profileDev = flag.Bool("profile_dev", false, "Enable development profiling")
```

Comment:
Can you make the comment describing this more specifically? For example, one might wonder "How does 'development profiling' differ from the Bazel --profile flag"

What would you think about renaming it to "log_timing_info" (I like green bikesheds)?

Reply:
The `profile_dev` naming was @alexeagle's preference so that it would be similar but different to the Bazel `profile` flag. This is a work in progress... plan is to have iBazel write build timing results to a user specified file and also open up a port to accept timing results from the browser as well to add to the results. Similar work is being done in parallel in iBlaze.